packages/apollos-church-api/src/data/content-item-slugs/data-source.js

```diff
@@ -0,0 +1,22 @@
+import RockApolloDataSource from '@apollosproject/rock-apollo-data-source';
+import ApollosConfig from '@apollosproject/config';
+
+const { ROCK } = ApollosConfig;
+
+export default class ContentChannelItemSlug extends RockApolloDataSource {
+  resource = 'ContentChannelItemSlugs';
+
+  getShareURL = async (id, contentChannelId, dataSources) => {
+    try {
+      const contentChannel = await dataSources.ContentChannel.getFromId(
+        contentChannelId
+      );
+      const slug = await this.request('ContentChannelItemSlugs')
+        .filter(`ContentChannelItemId eq ${id}`)
+        .get();
```

Comment:
You can use `.first` here :D

Reply:
I changed this to use `.first()`. Which was interesting because then I didn't have the array that I needed to choose the first one off of. Which meant I could change `slug[0].slug` to just `slug.slug`. I could probably even simplify that further if I needed to.